AnthonyGW/Bucketlist-FrontEnd
# #151515982 enhance UI and implement feedback

#### What does this PR do?
Improve on the UI made in development
Implement feedback received in previous PullRequests, Code Review and Code Defense.

#### Description of Task to be completed?
Use bootstrap to style the pages and forms.
Feedback Implemented:
1. Add comments to code.
2. Break long lines
3. Add newline to end of files
4. Remove unused imports

#### How should this be manually tested?
There are no test files implemented yet.
Test functionality by interacting with the GUI forms and buttons, the described routes have respective links to go to them:
1. On the base route (`/`), Click on the button `Register` to use create a new user by providing a username, email address and password in the form provided. App should open the Login form when successful.
2. Log in with account email address and password. App should redirect to user dashboard when successful.
3. Create a new bucketlist with the form to the left of the page.
4. View the bucketlists created to the right of the page.
5. Search for bucketlists by name using the search bar.
6. View the rest of the user's bucketlists (when they are more than 10) using the page navigation buttons.
7. Edit or delete a specific bucketlist using the links in the same row. User can leave the form fields blank to keep the same information when editing.
8. View a specific bucketlist's items by clicking on its name.
9. Create a new bucketlist item using the form to the left of the page.
10. View bucketlist items to the right of the page.

#### Any background context you want to provide?
The UI is based on the challenge 2 bootcamp project which can be found here: [BucketListUltima](https://github.com/AnthonyWaithaka/Bucket-List-Ultima)

Pivotal Tracker story: <https://www.pivotaltracker.com/story/show/151513228>

#### Screenshots
<img width="1280" alt="screen shot 2017-10-03 at 16 26 58" src="https://user-images.githubusercontent.com/31277260/31127540-f674188e-a857-11e7-8647-654683f3536e.png">

<img width="910" alt="screen shot 2017-10-03 at 16 27 07" src="https://user-images.githubusercontent.com/31277260/31127565-081ac3f8-a858-11e7-9f16-246d990fbb54.png">

<img width="858" alt="screen shot 2017-10-03 at 16 27 17" src="https://user-images.githubusercontent.com/31277260/31127592-191cda88-a858-11e7-95ce-44869ea208f4.png">

<img width="1256" alt="screen shot 2017-10-03 at 16 27 36" src="https://user-images.githubusercontent.com/31277260/31127619-2a018b50-a858-11e7-91ce-bc678a1d74a7.png">

<img width="1243" alt="screen shot 2017-10-03 at 16 28 22" src="https://user-images.githubusercontent.com/31277260/31127734-570aaaf0-a858-11e7-9cf9-37fb6195a353.png">
